docs/roadmap.md

```diff
@@ -0,0 +1,53 @@
+# Roadmap
+
+This section elaborates on proposed new features or tasks which are expected to
+be added to the product in the  foreseeable future. There should be no
+assumption of a commitment to deliver these features on specific dates neither
+in the given order. The development team will be doing their best to follow the
+proposed dates and priorities, but please bear in mind that plans to work on a
+given feature or task may be revised. All information is provided as a general
+guideline only and this section may be revised to provide newer information at
+any time.
+
+This product is an Incubated FIWARE Generic Enabler.  If you would like to learn
+about the overall Roadmap of FIWARE, please check section "Roadmap" on the
+[FIWARE Catalogue](https://github.com/Fiware/catalogue).
+
+## Short term (next release of the product)
+
+The following list of features are planned to be addressed in the short term,
+and incorporated in the next release of the product planned for February 2019.
+
+For more details, please have a look at the [next github milestones](https://github.com/smartsdk/ngsi-timeseries-api/milestone/2)
+and its associated issues. It is planned for February 15, 2019.
+
+- [GH #101](https://github.com/smartsdk/ngsi-timeseries-api/issues/101) will
+focus on supporting Virtual Entities (think of database views).
+- [GH #93](https://github.com/smartsdk/ngsi-timeseries-api/issues/93) will
+focus on supporting different attributes to be chosen to act as `time_index`.
+
+## Medium term (9 months following next release)
+
+The following list of features are planned to be addressed in the medium term,
+typically within the subsequent release(s) generated in the next 9 months after
+next planned release.
+
+For more details, please have a look at the [github milestones](https://github.com/smartsdk/ngsi-timeseries-api/milestones?direction=asc&sort=due_date&state=open)
+and their associated issues.
+
+- [Version 0.7](https://github.com/smartsdk/ngsi-timeseries-api/milestone/4)
+will focus on features to enhance the coupling of QL with visualization tools
+(like automating creation of datasources or graphs). It is planned for April
+2019.
+- [Version 1.0](https://github.com/smartsdk/ngsi-timeseries-api/milestone/6)
+will focus on completing the implementation of the original API endpoints,.
```

Comment:
things we could include here (then it may go beyond the 9 months, but who knows):
. cors support
- api for configuration
- improve array support
- support simple geo shapes in reply beside geo:json
- quantumleap as context broker datasource